packages/core/lib/console-child.js

```diff
@@ -9,9 +9,10 @@ const inputStrings = input[1];
 
 //detect config so we can get the provider and resolver without having to serialize
 //and deserialize them
-const detectedConfig = Config.detect({ network: yargs(input[0]).network });
+const detectedConfig = Config.detect({ network: yargs(input[0]).argv.network });
 const customConfig = detectedConfig.networks.develop || {};
-
+// console.log("custom config? " + JSON.stringify(customConfig));
+// console.log("detected config " + JSON.stringify(detectedConfig));
```

Comment:
These should come out.

Reply:
Oops! That's what I get for trying to get in a quick fix. :) Thanks!